doc/xml/release.xml

```diff
@@ -94,6 +105,17 @@
 
             <release-doc-list>
                 <release-improvement-list>
+                    <release-item>
+                        <commit subject="Add hint and documentation for protocol version mismatches."/>
+
+                        <release-item-contributor-list>
+                            <release-item-ideator id="loop-evgeny"/>
+                            <release-item-reviewer id="cynthia.shang"/>
+                        </release-item-contributor-list>
+
+                        <p>Add note that <backrest/> versions must match when running remotely.</p>
```

Comment:
Why is this section in here twice (lines 74 - 83 as well as here)?

Reply:
This commit changes both the core code and the docs so it gets two release notes.